syzygy/management/commands/makemigrations.py

```diff
@@ -23,9 +21,13 @@ def handle(self, *args, disable_syzygy, **options):
         # Monkey-patch makemigrations.MigrationAutodetector since the command
```

Comment:
you used partial before as the according to the comments, the class wasn't inheritable.
I take it that the partial object was "the monkey-patch" approach.
if that's the case, then comment is also can be removed